apps/fluent-tester/src/FluentTester/theme/CustomThemes.ts

```diff
@@ -13,7 +13,7 @@ export const lightnessOptions = [
 ];
 
 export class TesterThemeReference extends ThemeReference {
-  private _themeName: ThemeNames = 'Fluent';
+  private _themeName: ThemeNames = 'Apple';
```

Comment:
Yes.. this is where I would like @FalseLobster 's input. This is intended to be the default theme on iOS/macOS, but it differs from the already-defined default theme that uses the Fabric Web Palette. I could pass the theme when we initialize FluentTester in each test app at index.ts, but I'm not sure if I need to change the picker options in the drop down then?

Reply:
An update here: 
Win32/windows CI fail because `FluentTester` takes a dependency on `apple-theme`, which takes a dependency on `react-native-macos`, which doesn't import cleanly into windows/win32. 

Ideally, FluentTester should not need to know about the apple theme, and we should just pass it in via the macOS test app. I have not yet found a way to do this, and will need to modify FluentTester to take that in as a prop or something like that...

Reply:
we can use metro's per-platform module resolution to change the behavior of our fluent theme.  you could imagine a fluentTheme.macos.ts that calls createAppleTheme() and a fluentTheme.win32.ts that calls createOfficeTheme() or whatever.

Reply:
I thought that the Fluent theme in this context was just a fancy name for the default theme though? To which it would then become having the default theme return the new apple theme if on ios or macos, which didn't seem like a good idea

Reply:
Platform specific files fixed this! Thanks again!